docs/src/index.md

```diff
@@ -25,54 +25,62 @@ julia> Pkg.add("OceanBioME")
 
 ## Running your first model
 
-As a simple example let's run a Nutrient-Phytoplankton-Zooplankton-Detritus (NPZD) model in a two-dimensional simulation of a buoyancy front:
+As a simple example lets run a Nutrient-Phytoplankton-Zooplankton-Detritus (NPZD) model in a two-dimensional simulation of a buoyancy front. This also requires Oceananigans, so we will add that first:
 
-```@example quickstart
-using OceanBioME
-using Oceananigans, Oceananigans.Units
+```julia
```

Comment:
Why did you change @example to simple code? That won’t work. Documenter wont run the code but rather will just print it.

Reply:
Ah I didn't mean todo that

Reply:
(There is a chance I missed something here?)

Reply:
Think its because I copied from readme

Reply:
Cool. Okie!